### ISSUE TITLE
Packaging and installing with poetry

### ISSUE THREAD

**Issue by kfk** [kfk/grizly#265]
I will look into this as it would allow us to easily package grizly with everything else and publish on pypy. I also think we should use semantic versioning so we start getting aligned on the right versions for grizly. This means we could have a workflow running on x version and another one running on y version.

**Comment by michalz92** [comment]
Yes, Poetry looks very good. Interesting to see if it works. We can also add it as a step to Github Actions/other CI tool we'll be using in the future.

**Comment by michalz92** [comment]
I've tested Poetry on my private stuff and it works well. I've combined it with [cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.0/) ([data science template](https://drivendata.github.io/cookiecutter-data-science/)). I think it would be great if we created a grizly cookiecutter that could be used as a default project structure/setup.
